src/main/java/environment/Player.java

```diff
@@ -0,0 +1,45 @@
+package environment;
+
+import java.util.HashMap;
+
+public class Player {
+    private static final int MAX_ITEM_LIMIT = 5;
+    private final String name;
+    private final HashMap<String, Item> items;
+
+    public Player(String name) {
+        this.name = name;
+        this.items = new HashMap<>();
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public String getItemList() {
```

Comment:
If I understand correctly this is a getter method of "items", then naming it as "getItems" makes more sense to me. Or if this method functions more like a print method (and a potential getter method that return an Item object will be implemented in the future), naming it as "printItemList()" to avoid confusion may be better?